_assets/stylesheets/_base.scss

```diff
@@ -10,6 +10,7 @@ $border-radius: 2px;
 $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 $headings-font-family: "brandon-grotesque";
+$font-family-sans-serif: "brandon-grotesque";
```

Comment:
I know this is in the design, but I'd rather keep the main text areas in the default font for Bootstrap and reserve Brandon Grotesque for headlines.